src/expand.ts

```diff
@@ -95,9 +95,9 @@ export interface ExpansionOptions {
  * @param opts Options for the expansion process
  * @returns A string with the variable references replaced
  */
-export async function expandString(tmpl: string, opts: ExpansionOptions) {
+export async function expandString(tmpl: string | null | undefined, opts: ExpansionOptions): Promise<string> {
     if (!tmpl) {
-        return tmpl;
+        return tmpl as string;
```

Comment:
`as` is not type coercion.  If we must return a string here, return `""` explicitly.

Reply:
OK, the intension is do template like things, input string, return string, input null return null, that's we want